src/platform/forms/components/review/PreSubmitSection.jsx

```diff
@@ -0,0 +1,79 @@
+// libs
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+// platform - forms - selectors
+import { preSubmitSelector } from 'platform/forms/selectors/review';
+
+// platform - forms-system components
+import { PreSubmitSection as DefaultPreSubmitSection } from 'platform/forms-system/src/js/components/PreSubmitSection';
```

Comment:
It's confusing to me to write a new `PreSubmitSection` and then import the old one. What am I missing?

Reply:
See above said copy pasta note. 😄